docs/development/testing.md

```diff
@@ -0,0 +1,66 @@
+<!--
+SPDX-FileCopyrightText: 2024 3mdeb <contact@3mdeb.com>
+
+SPDX-License-Identifier: CC-BY-SA-4.0
+-->
+
+# Testing
+
+## Overview
+
+This document presetns the current status and progress of testing the TwPM
+module features. The tests used here are located in the
+[Dasharo OSFV repository](https://github.com/Dasharo/open-source-firmware-validation/blob/main/dasharo-security/tpm2-commands.robot).
+
+## Tests results
+
+The latest results (as of 11/01/2024) can be found
+[here](/test-results/2024_01_11_orange_crab_without_create_primary.html).
+
+## Hardware setup
+
+### Hardware list
+
+* [Protectli VP4670](https://docs.dasharo.com/variants/protectli_vp46xx/overview/)
+    - The TwPM is connected to LPC TPM header on this board, so we can test the
+      TPM features of the TwPM
+* [Orange Crab](https://github.com/orangecrab-fpga/orangecrab-hardware)
+    - The TwPM is implemented on this board
+
+### Connection to the platform
+
+Follow the [mainboard connection tutorial](/tutorials/mainboard-connection.md).
+
```

Comment:
```suggestion
Follow the [mainboard connection tutorial](/tutorials/mainboard-connection.md).
Make sure that LCLK and LAD lines aren't directly next to each other (e.g.
separate those with GND), otherwise inter-signal noise would cause bad reads.

```

Reply:
c2eee630c2901ca51fafb59d83fcd359bd679cad